packages/@aws-cdk/aws-ecs/lib/scheduled-ecs-task.ts

```diff
@@ -0,0 +1,98 @@
+import events = require('@aws-cdk/aws-events');
+import cdk = require('@aws-cdk/cdk');
+import { ICluster } from './cluster';
+import { ContainerImage } from './container-image';
+import { Ec2EventRuleTarget } from './ec2/ec2-event-rule-target';
+import { Ec2TaskDefinition } from './ec2/ec2-task-definition';
+import { AwsLogDriver } from './log-drivers/aws-log-driver';
+
+export interface ScheduledEc2TaskProps {
+  /**
+   * The cluster where your service will be deployed.
+   */
+  readonly cluster: ICluster;
+
+  /**
+   * The image to start.
+   */
+  readonly image: ContainerImage;
+
+  /**
+   * Number of desired copies of running tasks.
+   *
+   * @default 1
+   */
+  readonly desiredTaskCount?: number;
+
+  /**
+   * The hard limit (in MiB) of memory to present to the container.
+   *
+   * If your container attempts to exceed the allocated memory, the container
+   * is terminated.
+   *
+   * @default 256
+   */
+  readonly memoryLimitMiB?: number;
+
+  /**
+   * The schedule expression used to configure the event rule.
+   */
+  readonly scheduleExpression?: string;
```

Comment:
What is the expected syntax for schedule Expression? In the README the example says `'rate(1 minute)'`, which seems not the most intuitive syntax.

Reply:
The schedulingExpressions are values that are provided by cloudwatch events: https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html